packages/block-library/src/audio/save.js

```diff
@@ -17,7 +21,13 @@ export default function save( { attributes } ) {
 					preload={ preload }
 				/>
 				{ ! RichText.isEmpty( caption ) && (
-					<RichText.Content tagName="figcaption" value={ caption } />
+					<RichText.Content
+						tagName="figcaption"
+						value={ caption }
+						className={ __experimentalGetElementClassName(
```

Comment:
Any change to a save function of a block need a deprecation to be aded to that block. See https://github.com/WordPress/gutenberg/issues/45153